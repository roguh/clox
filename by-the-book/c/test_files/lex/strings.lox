"abcdef" '123' "123456\"" "multi
line
string!"
"after multiline?"
'abc'
