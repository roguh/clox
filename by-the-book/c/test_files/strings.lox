"abcdef" '123' "123456\""
