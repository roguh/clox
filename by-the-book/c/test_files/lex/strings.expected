   1:8    '"abcdef"'
   1:9    ';'
   1:15   ''123''
   1:16   ';'
   1:27   '"123456\""'
   3:8    '"multi
line
string!"'
   3:9    ';'
   4:18   '"after multiline?"'
   4:19   ';'
   5:5    ''abc''
   5:6    ';'
   5:6    EOF
